.ci/Jenkinsfile

```diff
@@ -521,9 +521,6 @@ def generateFunctionalTestStep(Map args = [:]){
     goArch = "arm64"
   }
 
-  // Skip scenarios per platform
-  tags += " && ~@skip:${goArch}"
```

Comment:
Removed because we do not need to skip by arch at the tag level, but at the `.e2e-testing.yaml` descriptor